rebar.config

```diff
@@ -81,7 +81,8 @@
          [
           {post_start,
            [
-            {wait_for_process, blockchain_worker}
+            {wait_for_process, blockchain_worker},
+            {pid, "miner.pid"}
```

Comment:
while it's nice to have a pidfile, shouldn't it be somewhere else, like `/tmp/`?

Reply:
depends on the os, but I think tmp is a great place.

Reply:
`/var/run` is canonical on BSD now. So, there are quite a few disagreeable options.